api/services/ocean/cd/schemas/verificationTemplate/oceanCDVerificationTemplateMetricCloudWatch.yaml

```diff
@@ -1,49 +1,64 @@
           properties:
             metric:
               type: object
+              description: The metric to return, including the metric name, namespace, and dimensions.
               properties:
+                metricName:
+                  type: string
+                  required: true
+                  description: The name of the metric.
+                namespace:
+                  type: string
+                  description: The namespace of the metric.
                 dimensions:
                   type: array
+                  description: A dimension is a name/value pair that is part of the identity of a metric. Because dimensions are part of the unique identifier for a metric, whenever you add a unique name/value pair to one of your metrics, you are creating a new variation of that metric. You can assign up to 30 dimensions to a metric.
                   items:
                     type: object
                     properties:
                       name:
                         type: string
+                        description: The name of the dimensions. These names must contain only ASCII characters, must include at least one non-whitespace character, and cannot start with a colon(':')
                       value:
                         type: string
-                metricName:
-                  type: string
-                namespace:
-                  type: string
-            expression:
-              type: string
+                        description: The value of the dimensions. These values must contain only ASCII characters and must include at least one non-whitespace character.
             period:
               type: integer
+              description: The granularity, in seconds, of the returned data points. For metrics with regular resolution, a period can be as short as one minute (60 seconds) and must be a multiple of 60. For high-resolution metrics that are collected at intervals of less than one minute, the period can be 1, 5, 10, 30, 60, or any multiple of 60. High-resolution metrics are those metrics stored by a PutMetricData call that includes a StorageResolution of 1 second.
             stat:
               type: string
+              description: The statistic to return. It can include any CloudWatch statistic or extended statistic.
             unit:
               type: string
-        period:
-          type: integer
+              description: This defines what unit you want to use when storing the metric. /n Examples:"Seconds|Microseconds|Milliseconds|Bytes|Kilobytes|Megabytes|Gigabytes|Terabytes|Bits|Kilobits|Megabits|Gigabits|Terabits|Percent|Count|Bytes/Second|Kilobytes/Second|Megabytes/Second|Gigabytes/Second|Terabytes/Second|Bits/Second|Kilobits/Second|Megabits/Second|Gigabits/Second|Terabits/Second|Count/Second|None"
```

Comment:
@AvishagK - This is a way to define "enum list type so it will show you examples of values